internal/gengapic/gengapic_test.go

```diff
@@ -499,3 +499,55 @@ func Test_isLRO(t *testing.T) {
 		}
 	}
 }
+
+func Test_transportParse(t *testing.T) {
+	for _, tst := range []struct {
+		param        string
+		expectedOpts options
+		expectErr    bool
+	}{
+		{
+			param: "transport=grpc,go-gapic-package=path;pkg",
+			expectedOpts: options{
+				transports: []Transport{grpc},
+				pkgPath:    "path",
+				pkgName:    "pkg",
+				outDir:     "path",
+			},
+			expectErr: false,
+		},
+		{
+			param: "transport=rest+grpc,go-gapic-package=path;pkg",
+			expectedOpts: options{
+				transports: []Transport{rest, grpc},
+				pkgPath:    "path",
+				pkgName:    "pkg",
+				outDir:     "path",
+			},
+			expectErr: false,
+		},
+		{
+			param:        "transport=tcp,go-gapic-package=path;pkg",
+			expectedOpts: options{},
+			expectErr:    true,
+		},
+		{
+			param: "go-gapic-package=path;pkg",
+			expectedOpts: options{
+				transports: []Transport{grpc},
+				pkgPath:    "path",
+				pkgName:    "pkg",
+				outDir:     "path",
+			},
+		},
+	} {
+		opts, err := ParseOptions(&tst.param)
+		if tst.expectErr {
```

Comment:
You should also check you don't get an error when you don't expect one. Maybe `if test.ExpectErr != (err!=nil)` ?

Reply:
Good point. Will add.